samtools.1

```diff
@@ -1432,6 +1432,25 @@ Specifies the output filename.  By default output is sent to stdout.
 .RE
 
 .TP \"-------- help etc
+.TP 8
```

Comment:
These appear to have moved to some odd location rather than the correct part of the man page.  I also don't see -1 listed anywhere.  Do you have an ealier copy of the man page? I suspect a merge broke somewhere.